opentelemetry-sdk/src/logs/log_emitter.rs

```diff
@@ -210,12 +205,10 @@ impl opentelemetry_api::logs::Logger for Logger {
         let config = provider.config();
         for processor in provider.log_processors() {
             let mut record = record.clone();
-            if self.include_trace_context {
-                let ctx = Context::current();
-                if ctx.has_active_span() {
-                    let span = ctx.span();
-                    record.trace_context = Some(span.span_context().into());
-                }
+            let ctx = Context::current();
+            if ctx.has_active_span() {
+                let span = ctx.span();
+                record.trace_context = Some(span.span_context().into());
```

Comment:
Now that we merged #1140. Maybe use `map_current` here?

Reply:
Have resolved the merge conflicts now, and using `map_current` for active context.